### ISSUE TITLE
Bug: 

### ISSUE THREAD

**Issue by husayngokal** [Qiskit/platypus#1525]
### Where does the issue happen?

https://learn.qiskit.org/course/introduction/describing-quantum-computers

### What is the content issue?

When you have an amplitude with a phase of 1 to 179 (making a "positive amplitude"), regardless of the magnitude, the amplitude's direction is upward:

<img width="696" alt="Positive Amplitude" src="https://user-images.githubusercontent.com/92259277/188563854-155d85c8-c4bd-461d-99ad-4144a1aac851.png">

And when you have an amplitude with a phase of 181 to 359 (making a "negative amplitude"), regardless of the magnitude, the amplitude's direction is downward:

<img width="696" alt="Negative Amplitude" src="https://user-images.githubusercontent.com/92259277/188564294-b2dc49e1-8a03-4387-a449-f6370d74f1c9.png">

I believe that the text in the image attached below should be changed to say "upward" and "downward" instead of "forward" and "backward". It just makes more sense that way :)

### Screenshots

<img width="694" alt="Amplitude Content Bug" src="https://user-images.githubusercontent.com/92259277/188562041-3a819078-8a1d-46a2-b3c7-2c695ac559c4.png">

**Comment by frankharkins** [comment]
Hi @husayngokal, thanks for bringing this up. I think there is some confusion here over 'positive' and 'negative'. In the first image (of the widget) I believe you're using a 'negative amplitude' to refer to an amplitude whose phase displays as negative on the widget?

In the 'Describing quantum computers' page, we choose to ignore all directions except 0° and 180° (which we call 'forward' and 'backwards'). The reason for choosing this is that amplitudes are actually complex numbers, and the arrows show positions on the [complex plane](https://en.wikipedia.org/wiki/Complex_plane). Since we wanted to avoid complex numbers for those who might not be familiar with them, we stick to real numbers, which constrains us to the X-axis on the complex plane.

I do see the confusion and I think it can be cleared up with a change of language in the paragraph you highlighted. I've changed the title to reflect this, but please let me know if you disagree.

**Comment by husayngokal** [comment]
Hey! First of all, I am not defining positive or negative amplitudes to refer to amplitude phases that display as a positive or negative on the widget. I am solely taking into account the direction that the arrow is pointing towards. Secondly, I believe that the decision to display positive and negative phases on the widget isn't arbitrary.

From what I've seen on the IBM Quantum Circuit Composer, as well as other resources online, the top half of the circle refers to positive values from +1 to 0 (read left to right), and the bottom half of the circle refers to negative values from 0 to -1 (read right to left). This is a sort of standardization that isn't explicitly explained anywhere, but is so common it might as well be convention. The widget also uses the exact same standardization that I just explained here, where positive amplitude phases point upward in the top half of the circle, and negative amplitude phases point downward in the bottom half of the circle.

Now, since we're only dealing with values on the x-axis of the complex plane, it would make sense to use the terms "forward" and "backward". But the first line of this paragraph is referring to the fact that any amplitude can be displayed as values ranging from +1 to -1. This includes values that need to be written as complex numbers.

So either, we remove this phrase to eliminate confusion, or we change the paragraph to use the terms "upwards" and "downwards" (like I propose), and mention at the end that we'll only be working with values that are constrained to the x-axis (hence "forward" and "backward"), just to be complete.

In short, this paragraph just needs a bit of language clarification (as you suggested), as well as some additional context.

**Comment by husayngokal** [comment]
Hey, @frankharkins! It's been quite a while - can we move forward with this PR?

**Comment by frankharkins** [comment]
Hi, sorry this got overlooked.

> From what I've seen on the IBM Quantum Circuit Composer...

If you're referring to the circles at the end of the qubit wires, they're different. Those circles represent qubit states (similar to a Bloch sphere), the circles in the images you posted represent amplitudes, which are just complex numbers. It's an unfortunate coincidence that they look similar

The circles are just positions on the complex plane

![](https://www.mathsisfun.com/algebra/images/euler-formula-circle.svg)

**Comment by husayngokal** [comment]
Ah, I see. Thank you for the clarification :)
